ocaml/tests/test_xapi_xenops.ml

```diff
@@ -87,7 +87,6 @@ let test_xapi_restart_inner () =
     Xapi_xenops.start ~__context ~self:vm5 false false ;
     Xapi_xenops.start ~__context ~self:vm6 false false ;
     Xapi_xenops.start ~__context ~self:vm7 false false ;
-    (* vm6 is a ntnx CVM *)
```

Comment:
Since we are removing CVM related logic as CVM is not supported now, is the test for vm6 still needed?